sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Release History
 
-## 5.1.0 (unreleased)
+## 5.1.0-beta.4 (unreleased)
 
-- A new option to control how the offset is calculated by the service, `stringIndexType`, is added to `analyzeSentiment`, `recognizeEntities`, `recognizePiiEntities`, and `beginAnalyzeHealthcareEntities`. Furthermore, `stringIndexType` is added to task types `EntitiesTask` and `PiiTask`, which are the types of input tasks to the `beginAnalyze` method. For more information, see [the Text Analytics documentation](https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/text-offsets#offsets-in-api-version-31-preview).
 - [Breaking] `beginAnalyzeHealthcare` is renamed to `beginAnalyzeHealthcareEntities`.
+- [Breaking] `beginAnalyze` is renamed to `beginAnalyzeBatchActions`.
+- A new option to control how the offset is calculated by the service, `stringIndexType`, is added to `analyzeSentiment`, `recognizeEntities`, `recognizePiiEntities`, and `beginAnalyzeHealthcareEntities`. Furthermore, `stringIndexType` is added to task types `EntitiesTask` and `PiiTask`, which are the types of input actions to the `beginAnalyzeBatchActions` method. For more information, see [the Text Analytics documentation](https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/text-offsets#offsets-in-api-version-31-preview).
```

Comment:
Do we want to call `EntitiesRecognitionAction` , `PiiEntitiesRecognitionAction`  and `KeyPhrasesExtractionAction`?

Reply:
Fixed.